apps/moderation/src/components/content-card/content-card.tsx

```diff
@@ -44,26 +44,27 @@ const ContentCard: React.FC<Omit<IContentProps, 'entryData'>> = props => {
     reviewDecisionLabel,
   } = props;
 
-  const profileDataReq = useGetProfile(entryId);
-  const profile = profileDataReq.data;
+  const profileDataQuery = useGetProfile(entryId, props.user, itemType === 'account');
+  const profile = profileDataQuery.data;
 
-  const postReq = usePost({ postId: entryId, enabler: !!entryId });
-  const commentReq = useComment(entryId);
+  const postQuery = usePost({ postId: entryId, enabler: itemType === 'post' });
+  const commentQuery = useComment(entryId, itemType === 'reply' || itemType === 'comment');
```

Comment:
aren't the item types declared somewhere already instead of manually specifying them?

Reply:
defined now...